src/cascadia/TerminalSettingsModel/CascadiaSettings.cpp

```diff
@@ -1156,6 +1156,18 @@ void CascadiaSettings::ExportFile(winrt::hstring path, winrt::hstring content)
 
 void CascadiaSettings::_validateThemeExists()
 {
+    if (_globals->Themes().Size() == 0)

```

Comment:
- If this is only hit in the tests, shouldn't we be adding the "system" theme in the tests?
- In a (possibly distant) world where we have multiple `GlobalAppSettings` in the inheritance structure, I could see some of them not having a theme set. Is that going to be a problem here where we add an empty "system" theme?

Reply:
The empty system theme _IS_ the system theme (that's actually how it works!)

I think it's important to have in the validation, honestly. defaults.json is a crutch, right?

Reply:
I'm rewording the comment here. 

I think it's fine to leave this as a validation step. In reality, we'll always have a `themes` in `defaults.json`, but oftentimes the tests just don't even bother with a `defaults`, for simplicity. 

The alternative would be to make the themes in `defaults.json` actually hardcoded instead, but that seemed more fragile. Seemed like it was a good idea to have the themes actually listed there (esp. for the `terminalBackground` treatment we're giving the tabs in #13554)